examples/text/main.go

```diff
@@ -48,6 +48,7 @@ func main() {
 			mgl32.Vec3{-400, -300 + vo, 0},
 			mgl32.Vec2{j * 8, j * 8},
 			color,
+			mgl32.Vec4{0, 0, 0, -.17},
```

Comment:
Any specific reason for the .17?

Reply:
no reason, just condensed enough for my liking

Reply:
Ah ok, at first I thought that was connected to the transparency issue